codemp/game/bg_saber.c

```diff
@@ -349,6 +348,38 @@ saberMoveData_t	saberMoveData[LS_MOVE_MAX] = {//							NB:randomized
 	{"Reflect LL",	BOTH_P1_S1_BL,		Q_R,	Q_BR,	AFLAG_ACTIVE,	50,		BLK_WIDE,	LS_R_TL2BR,		LS_A_BR2TL,		300	},	// LS_PARRY_LL,
 };
 
+void BG_FixSaberMoveData(void) {
+#if defined(_GAME)
+	const qboolean doFix = !!g_fixSaberMoveData.integer;
+#elif defined(_CGAME)
+	const char *cs = CG_ConfigString(CS_LEGACY_FIXES);
+	const uint32_t legacyFixes = strtoul(cs, NULL, 0);
+	const qboolean doFix = !!(legacyFixes & (1 << LEGACYFIX_SABERMOVEDATA));
+#endif
+	saberMoveData_t *move;
+
+	for (move = saberMoveData; move - saberMoveData < ARRAY_LEN(saberMoveData); move++) {
+		if (!strcmp(move->name, "BParry LR")) {
+			move->animToUse = doFix ? BOTH_H1_S1_BR : BOTH_H1_S1_BL;
```

Comment:
Maybe just return early on !doFix instead?

Reply:
Nope, since the server can currently toggle it back and forth. We want to be able to restore the original value.